### PR TITLE
feat(avo-2187): integrate admin-core-api into hetarchief proxy

### PR DESCRIPTION
**Pull request by bertyhell**
https://meemoo.atlassian.net/browse/AVO-2187

requires admin-core PR: https://github.com/viaacode/react-admin-core-module/pull/48

### CODE REVIEW

**Review by WardVerc1 — APPROVED** [review]

File: .env/.env.template [hunk 1]
```diff
@@ -11,11 +11,9 @@ CLIENT_API_KEY=
 CORS_ENABLE_WHITELIST=false
 
 # GRAPHQL
-GRAPHQL_URL=http://localhost:9000/v1/graphql
-GRAPHQL_SECRET=myGraphQlSecret
+GRAPHQL_URL_HET_ARCHIEF=http://localhost:9000/v1/graphql
+GRAPHQL_SECRET_HET_ARCHIEF=myGraphQlSecret
```
Comment (ian-emsens-shd) [inline .env/.env.template:15]: Why does this need to be namespaced in the implementing application? The admin core will only ever require one set of url/secret, it'll never need both `GRAPHQL_URL_HET_ARCHIEF` and `GRAPHQL_URL_AVO`.

File: src/config/config.types.ts [hunk 2]
```diff
@@ -1,19 +1,16 @@
+import { AvoOrHetArchief } from '@meemoo/admin-core-api';
```
Comment (ian-emsens-shd) [inline src/config/config.types.ts:1]: Really need to move away from these naming schemes, they're not clear nor scalable when additional applications are added.

**Review by ian-emsens-shd — APPROVED** [review]

File: .env/.env.template (hunk 1 above)
Comment (bertyhell) [reply .env/.env.template:15]: there are multiple graphql databases:
* org api
* logging api
* hetarchief
* avo

This suffix is to make a clear difference between the hetarchief database and the logging database.

File: src/config/config.types.ts (hunk 2 above)
Comment (bertyhell) [reply src/config/config.types.ts:1]: agreed, in the future we might want to go towards: applicationName: 'avo' | 'hetarchief' | '...'
for now this name is clear, but not scalable.

**Pull request merged**